src/components/Tree/readme.md

```diff
@@ -79,10 +90,13 @@
         }
     ];
 
+    const selectedNode = '';
+
     import React, { useState, useEffect } from 'react';
 
     const TreeExample = () => {
         const [data, setData] = React.useState(initialData);
+        const [node, setSelectedNode] = React.useState(selectedNode);
```

Comment:
since the import above has useState, then here you can remove `React.`:
```const [node, setSelectedNode] = useState(selectedNode);```
same for the data state